### PR TITLE
Add hotkey for help about advanced hotkeys (bsc#1010039)

### PR DESCRIPTION
**Pull request by cwh42**
hotkey-help-ncurses
It was not possible to use Shift-F1 as an universal hotkey-helper-hotkey in ncurses. It depends on the terminal whether it reaches the application, or not.
So I added two hotkeys:
 - Shift-F1, which only works on a linux console (seen by the code as F13)
 - Ctrl-D F1, working even in other terminals, like xterm.

I chose Ctrl-D as a prefix, because most of the other advanced hotkeys also do so.

### CODE REVIEW

File: src/NCDialog.cc [hunk 1]
```diff
@@ -1083,6 +1084,10 @@ void NCDialog::processInput( int timeout_millisec )
 
 		break;
 
+	    case KEY_F( 13 ):
```
Comment (shundhammer) [inline src/NCDialog.cc]: Please add a comment that says that this is the equivalent to Shift-F1 for certain $TERM types (linux? xterm?).

**Comment by shundhammer** [comment]
Besides the missing comment above, LGTM.

**Comment by cwh42** [comment]
Comment is there, now.

**Comment by shundhammer** [comment]
LGTM

**Pull request merged**